test/e2e/chaos/networkchaos/misc.go

```diff
@@ -241,8 +240,6 @@ func couldConnect(c http.Client, sourcePort uint16, targetPodIP string, targetPo
 		return false
 	}
 
-	time.Sleep(2 * time.Second)
```

Comment:
This line was originally come from [this PR L2180](https://github.com/chaos-mesh/chaos-mesh/pull/1069/files#diff-6370badef8e5715e4f72fad6d6488b517cd1cca6f98764aa93209e5d3710e36bR2180), see related [PR #1069 comment](https://github.com/chaos-mesh/chaos-mesh/pull/1069#issuecomment-717027901).

Reply:
thanks, I will take a look